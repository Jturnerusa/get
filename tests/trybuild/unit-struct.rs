use get::Get;

#[derive(Get)]
struct Cat;

pub fn main() {}
