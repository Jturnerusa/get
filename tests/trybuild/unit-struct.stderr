error: proc-macro derive panicked
 --> tests/trybuild/unit-struct.rs:3:10
  |
3 | #[derive(Get)]
  |          ^^^
  |
  = help: message: called `Result::unwrap()` on an `Err` value: "can not generate getters on a unit struct"
